plugins/modules/site_workflow_manager.py

```diff
@@ -482,29 +567,126 @@ def site_exists(self):
           'get_site' function in the 'sites' family. It utilizes the
           'site_name' parameter from the 'want' attribute to identify the site.
         """
-
         site_exists = False
         current_site = {}
         response = None
+        if self.get_ccc_version_as_integer() <= self.version_2_3_5_3:
+            response = self.get_site_id(self.want.get("site_name"))
+            if response:
+                response = response.get("response")
+                self.log("Received API response from 'get_site': {0}".format(
+                    str(response)), "DEBUG")
+                current_site = self.get_current_site(response)
```

Comment:
Add the check if you receive current_site details then only we can say that site_exists and return site details.

Reply:
Added check for "currect_site"